src/lib/code-sample/javascript.ts

```diff
@@ -0,0 +1,18 @@
+import { camelCase } from 'change-case'
+
+import type { CodeSampleDefinition } from './schema.js'
+
+export const createJavascriptRequest = (
+  request: CodeSampleDefinition['request'],
+): string => {
+  const parts = request.path.split('/')
+  return `await seam.${parts.map((p) => camelCase(p)).join('.')}(${JSON.stringify(request.parameters)})`
+}
+
+export const createJavascriptResponse = (
+  response: CodeSampleDefinition['response'],
+): string => {
+  const { body } = response
+  if (body == null) return 'void'
+  return JSON.stringify(body)
```

Comment:
Does it make sense to return stringified body here if our JS SDK doesn't return the top level resource key of the response but the resource object itself?

Reply:
No it matche the SDK, I just missed that.